OWNERS.yaml

```diff
@@ -1,3 +1,7 @@
+# PLEASE DO NOT REMOVE - For an explanation of the OWNERS.yaml rules and syntax,
```

Comment:
IMO, the `PLEASE DO NOT REMOVE` is distracting. WDYT about re-wording the comment to the following?

```
# For an explanation of the OWNERS.yaml rules and syntax, see:
# https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example.yaml
```